### PR TITLE
make status column consistent

### PR DESCRIPTION
**Pull request by malaffoon**

### CODE REVIEW

File: webapp/src/main/webapp/src/app/student/component/student-exam-table/student-exam-table.component.html [hunk 1]
```diff
@@ -167,7 +167,10 @@
 
         <div *ngSwitchCase="'status'">
           <span
-            *ngIf="rowData.exam.administrativeCondition"
+            *ngIf="
+              rowData.exam.administrativeCondition &&
+              rowData.exam.administrativeCondition !== 'Valid'
```
Comment (malaffoon) [inline webapp/src/main/webapp/src/app/student/component/student-exam-table/student-exam-table.component.html]: make it consistent with the group/grade display (results-by-student.component)

Comment (esotrope) [reply webapp/src/main/webapp/src/app/student/component/student-exam-table/student-exam-table.component.html]: 👍 what about `SD`?

Comment (malaffoon) [reply webapp/src/main/webapp/src/app/student/component/student-exam-table/student-exam-table.component.html]: Just mimicking the other table.

Comment (esotrope) [reply webapp/src/main/webapp/src/app/student/component/student-exam-table/student-exam-table.component.html]: 👍

**Pull request merged**